port_ocean/core/event_listener/polling.py

```diff
@@ -83,6 +106,8 @@ async def resync() -> None:
                 signal_handler.register(running_task.cancel)
 
                 await running_task
+                await self.after_resync()
+                self.is_first_resync = False
```

Comment:
Why is this needed?

Reply:
This is only used for the resync state update
since polling listener is running a resync automatically only on installation after that all triggers are manual.